MultisigIntegrationTests/TokenRegistryIntegrationTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TokenRegistryIntegrationTests.swift
+//  MultisigIntegrationTests
+//
+//  Created by Dmitry Bespalov on 13.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
+@testable import Multisig
+
+class TokenRegistryIntegrationTests: XCTestCase {
+
+    let registry = TokenRegistry()
+
+    func testLoad() {
+        XCTAssertNil(registry["0x16baF0dE678E52367adC69fD067E5eDd1D33e3bF"])
```

Comment:
Please define shared store for those addresses instead of using the hardcoded ones

Reply:
ok

Reply:
now I read it again and not sure what you mean - to extract addresses in constants?

Reply:
yes